demo/chbench/mzcompose.yml

```diff
@@ -42,6 +42,22 @@ services:
       # 1000 was chosen by fair dice roll
       - DIFFERENTIAL_EAGER_MERGE=1000
       - DEFAULT_PROGRESS_MODE=${DEFAULT_PROGRESS_MODE}
+  materialized-release:
+    image: materialize/materialized:${MATERIALIZED_RELEASE_TAG:-v0.5.3}
```

Comment:
I think it would be nice to allow mzbuild to understand repos/tags in addition to just named projects. @benesch is there a philosophical reason to not do that, or is it just work that hasn't happened yet?